sass/main.scss

```
@@ -637,20 +636,28 @@ footer {
     overflow-x: clip;
 
     #footer-content {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-around;
+        
         width: 400px;
-        height: 240px;
+        height: 235px;
         margin: auto;
-        padding: 30px;
+        padding: 15px;
         text-align: center;
 
-        p, a, i {
-            margin-bottom: 10px;
+        h2 {
+            margin-bottom: -5px;
+        }
+        
+        p {
+            font-size: 0.95em;
         }
 
         // FA brand icons
         i {
             font-size: 3em;
-            padding-right: 10px;
+            padding: 0 5px;
```

Good catch on the padding here! Although style-wise, lets add units to the `0` to keep it inline with the `px` on the right:
```suggestion
            padding: 0px 5px;
```
(rest of stylesheet follows this convention, aside when unit is undefined, like for `marign: 0 auto;`)